tests/worker/test_workflow.py

```diff
@@ -235,12 +236,27 @@ async def test_workflow_info(client: Client, env: WorkflowEnvironment):
         )
         assert uuid.UUID(info["run_id"]).version == 7
         assert info["run_timeout"] is None
-        datetime.fromisoformat(info["start_time"])
         assert info["task_queue"] == worker.task_queue
         assert info["task_timeout"] == "0:00:10"
         assert info["workflow_id"] == workflow_id
         assert info["workflow_type"] == "InfoWorkflow"
 
+        async for e in handle.fetch_history_events():
```

Comment:
Can do `history.fetch_history` if you want an array of all events instead of the async iterator, but doesn't matter much